docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -92,7 +92,7 @@ Play's rewind operation currently does not support the following elements:
 - Play rewinds to an element, not to an element instance. For example, if you wanted to rewind your process to a sequential multi-instance service task which ran five times, it will rewind your process to the first instance of that service task.
 - Play rewinds processes by initiating a new instance and executing each element. However, if any element behaves differently from the previous execution, such as a Connector returning a different result, the rewind may fail.
 
-## Scenarios
+## Scenarios <span class="badge badge--cloud">Camunda 8 SaaS only</span> {#scenarios}
```

Comment:
I don't think we need to include this in the title as well, we already mention it in the limitations. wdyt?

Reply:
ok then 😄

Reply:
I just noticed there is a preview environment and it looks much better than the state I reviewed it 😄 
<img width="449" alt="Screenshot 2025-04-17 at 15 27 14" src="https://github.com/user-attachments/assets/40a8b8f0-6e9e-42b8-b6fb-6423d4679fd6" />
<img width="569" alt="Screenshot 2025-04-17 at 15 27 26" src="https://github.com/user-attachments/assets/3062039b-7852-465a-9a4a-b0963dedf5e9" />